lib/utils/cert/selfsigned.go

```diff
@@ -87,6 +87,10 @@ func GenerateSelfSignedCert(hostNames []string) (*Credentials, error) {
 	if ips != nil {
 		template.IPAddresses = append(ips, net.ParseIP("::1"))
 	}
+	for _, ip := range ipAddresses {
+		template.IPAddresses = append(template.IPAddresses, net.ParseIP(ip))
```

Comment:
Should we handle the case where `ParseIP` returns nil?

Reply:
@zmb3 add error handling